go.mod

```diff
@@ -52,12 +52,13 @@ require (
 	github.com/spf13/cobra v1.8.1 // indirect
 	github.com/x448/float16 v0.8.4 // indirect
 	go.opentelemetry.io/auto/sdk v1.1.0 // indirect
+	golang.org/x/mod v0.23.0 // indirect
 	gopkg.in/evanphx/json-patch.v4 v4.12.0 // indirect
 )
 
 require (
 	github.com/asaskevich/govalidator v0.0.0-20230301143203-a9d515a09cc2 // indirect
-	github.com/aws/aws-sdk-go v1.55.6
+	github.com/aws/aws-sdk-go v1.55.5
```

Comment:
should we downgrade?

Reply:
No we should not, this change is mine and not related to the upstream, need to remove it.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1094/commits/f0dab04aeb8b4aebe9423402a17269095cb6b1af